extensions/amp-consent/0.1/consent-config.js

```diff
@@ -150,7 +152,40 @@ export class ConsentConfig {
       }
     }
 
-    return config;
+    return this.mergeGeoOverride_(config).then(() => config);
+  }
+
+  /**
+   * Merge correct geoOverride object into toplevel config, then
+   * validate.
+   * @param {JsonObject} config
+   * @return {!Promise}
+   */
+  mergeGeoOverride_(config) {
```

Comment:
to follow the the immutability principal, let's clone the config 
```
const mergedConfig = Object.assign({}, config);
```

Reply:
Sounds good.